### PR TITLE
Update Export instruction. Update portal upgrade.

### PR DESCRIPTION
**Pull request by anartovich**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Two changes:
1. Wrong click sequence for PS to export a policy package
2. There was an unneeded step in API Portal upgrade process, and I've added clarification for bumping the file size limit for uploading the Joomla package (not applicable!)

## Deploy preview link

https://deploy-preview-2385--axway-open-docs.netlify.app/docs/apim_installation/apiportal_install/upgrade_automatic/#upgrade-from-the-joomla-administrator-interface

### CODE REVIEW

File: content/en/docs/apim_installation/apiportal_install/upgrade_automatic.md [hunk 1]
```diff
@@ -46,12 +46,8 @@ If you have a **7.5.5** or **7.6.2** API Portal installation, you can upgrade di
    ```
    tar xpzvf <package_name>.tgz
    ```
-3. Give executable permissions to the script `apiportal_cumulative_upgrade.sh`:
```
Comment (dginevaAxway) [inline content/en/docs/apim_installation/apiportal_install/upgrade_automatic.md:49]: I consulted with Aleks Stoyanov and he confirmed that this step might be required depending on how to customer extract the script. Execution of this step won't harm so I suggest to keep it there. The rest seems fine to me.
cc: @astoyanovAxway

File: content/en/docs/apim_installation/apiportal_install/upgrade_automatic.md [hunk 2]
```diff
@@ -67,15 +63,25 @@ If you have a 7.7.x API Portal installation, you can upgrade to the latest versi
...
 4. Log in to the Joomla! Administrator Interface (JAI) (`https://<API Portal host>/administrator`).
 5. Click **Components > Joomla! Update**, and click the **Upload & Update** tab. If **Joomla! Update** is not visible in the menu, connect to your user database and execute the following query for API Portal database:
 
    ```
    update s8f7h_menu set menutype='main' where title like 'com_joomlaupdate'
    ```
+
 6. Select the Joomla! upgrade package (for example, `joomla-update-package-3.9.14-package.zip`) from your file system.
 7. Click **Upload & Install**, and follow the displayed instructions.
+
```
Comment (dginevaAxway) [inline content/en/docs/apim_installation/apiportal_install/upgrade_automatic.md:77]: @anartovich  what is the reason for this change, joomla 3.9.x didn't hit the limits, Joomla 4 hits it and then you need to follow next section in this documentation to upgrade to May 2022 where this is covered in step 3:

> 3 Create an apiportal.ini file inside the additional PHP settings directory with the following content:
> 
> post_max_size = 60m
> upload_max_filesize = 60m

**Comment by anartovich** [comment]
thank you, @dginevaAxway ! Actually, if I execute the command as written, I don't need the step I've added. But I didn't copy/paste the command, I just type the command as I got used to with tar. So, I didn't use the "-p" flag that protects permission of the packaged files. That's why I've added this line. I guess some people may do the same mistake

File: content/en/docs/apim_installation/apiportal_install/upgrade_automatic.md (hunk 2 above)
Comment (anartovich) [reply content/en/docs/apim_installation/apiportal_install/upgrade_automatic.md:77]: this is what I see in the RefEnv when I work with a fresh install of Portal. I didn't run the upgrade to May 2022. So, I didn't know about the change.
When you say that Joomla 3.9.x didn't hit the limit, what do you mean? The limit on the file size upload is not controlled by Joomla. It is part of the php.ini. Am I wrong?

Comment (dginevaAxway) [reply content/en/docs/apim_installation/apiportal_install/upgrade_automatic.md:77]: hm, what do you mean fresh install, you are changing this in upgrade section in the doc. On install this limit should be handled by updated php.ini file. I mean that joomla 4 package get bigger and reach the limit for file upload, so need so be increased as you suggested, but for Joomla 4 this is covered by next section, while this one is for versions before may 2022. Are you using some pre- build script for install?

Comment (andreamussap) [reply content/en/docs/apim_installation/apiportal_install/upgrade_automatic.md:77]: Just a note while I wait for you to finish this discussion - About "_when I work with a fresh install of Portal_" --> This page is about upgrading the software. Customers who are doing a fresh install should read "Install API Portal" first. So, the instructions on the upgrade page should reflect what to do in case of upgrade, not first install.

Comment (anartovich) [reply content/en/docs/apim_installation/apiportal_install/upgrade_automatic.md:77]: "fresh install" - I meant that in our RefEnv we already changed that file size limit number, so when I upgrade Portal, the updated number is shown in the Joomla interface. But when I got a VM with API Portal installed, and I was doing an upgrade, the file size limit was 2 MB, so I had to change it. I can't tell you for sure what API Portal version it was. If this is fixed with the latest release of API Portal, then I'm OK to remove my update.

Comment (dginevaAxway) [reply content/en/docs/apim_installation/apiportal_install/upgrade_automatic.md:77]: yes, from May 2022 this should be handled for fresh install, for upgrade to May 2022 we added in the steps in completely new section "Upgrade to API Portal May 2022 release" plus few other steps, so I think won't be needed for previews versions.

Comment (kvedenskii-axway) [reply content/en/docs/apim_installation/apiportal_install/upgrade_automatic.md:77]: @anartovich 
There is probably something wrong with your VM. The 2 settings in question are overriden by `apiportal-joomla*.rpm` package which is a part of API Portal installation package as seen from an extract of installation log here (2 last lines alter php.ini file)
```
   1:apiportal-joomla-7.7.20220228-1  ################################# [100%]
start configuring joomla
Wed Aug 10 17:01:20 EEST 2022
Restore joomla database.
Enable large file size upload in apache php.
Disable output_buffering in php.
```
It used to be 20m (since 2016 if git doesn't lie), and it's changed to 40m with API Portal 2022 May release

File: content/en/docs/apim_installation/apiportal_install/upgrade_automatic.md (hunk 1 above)
Comment (dginevaAxway) [reply content/en/docs/apim_installation/apiportal_install/upgrade_automatic.md:49]: @anartovich  did we agree that this should stay in order to prevent some users missing the right permissions, since I am not exactly sure how to understand your comment about this?

File: content/en/docs/apim_installation/apiportal_install/upgrade_automatic.md (hunk 2 above)
Comment (anartovich) [reply content/en/docs/apim_installation/apiportal_install/upgrade_automatic.md:77]: OK, thank you for your response, @kvedenskii-axway ! I will review it. But in any case, I see that it's been handled in the new release of API Portal, so my request for the update can be canceled.

Comment (dginevaAxway) [reply content/en/docs/apim_installation/apiportal_install/upgrade_automatic.md:77]: I assume this was typo :) 2mb-20mb or the way the reference environment is installed. Anyway if is needed we can take this offline @anartovich

File: content/en/docs/apim_installation/apiportal_install/upgrade_automatic.md (hunk 1 above)
Comment (anartovich) [reply content/en/docs/apim_installation/apiportal_install/upgrade_automatic.md:49]: yes, we agree. Let's keep this addition.

File: content/en/docs/apim_installation/apiportal_install/upgrade_automatic.md (hunk 2 above)
Comment (andreamussap) [reply content/en/docs/apim_installation/apiportal_install/upgrade_automatic.md:83]: @anartovich based on your comment "my request for the update can be canceled.", I'll close this pull request, ok?

File: content/en/docs/apigtw_devops/promotion_example.md [hunk 3]
```diff
@@ -115,7 +115,7 @@ Alternatively, you can environmentalize a certificate using an alias. For exampl
 
 The policy developer finishes editing and environmentalizing the configuration that they are running with, and deploys it to the API Gateway. They must then save the policy package in Policy Studio to enable promotion to the testing environment. To save the policy package, perform the following steps:
 
-1. When the active configuration is loaded, select **File > Save > Policy Package**.
+1. When the active configuration is loaded, select **File > Export > Policy Package**.
```
Comment (dginevaAxway) [inline content/en/docs/apigtw_devops/promotion_example.md:118]: @andreamussap , this is not for API Portal and I can't confirm it, but the rest for API Portal can be canceled. @anartovich ?

Comment (anartovich) [reply content/en/docs/apigtw_devops/promotion_example.md:118]: @dginevaAxway , yes those comments that I added about the file size for Joomla can be canceled. And I can confirm that this change with File -> Export -> Policy Package is correct. It doesn't depend on any specific condition. It is just part of the Policy Studio interface.

Comment (andreamussap) [reply content/en/docs/apigtw_devops/promotion_example.md:118]: Ah, ok, so the part around Policy studio does not concern to API Portal team. I'll check which team can review this. Thanks

Comment (andreamussap) [reply content/en/docs/apigtw_devops/promotion_example.md:118]: @JohnD-Axway could you please review/approve this change?

**Review by Bcolemanaxway — APPROVED** [review]

File: content/en/docs/apigtw_devops/promotion_example.md (hunk 3 above)
Comment (Bcolemanaxway) [reply content/en/docs/apigtw_devops/promotion_example.md:118]: @andreamussap Looks good to me
